snapshots.sh

```
@@ -29,34 +29,57 @@ Commands:
-      echo "Checking snapshot tests"
-      helm chartsnap -c "./charts/$chart" -f "./$values_dir/$chart"
-      ;;
-    *)
-      usage
-      exit 1
-      ;;
-  esac
+  local charts=("wandb-base" "operator-wandb")
+  local values_dir="test-configs"
+
+  if [ $# -gte 2 ] && [ -n "$2" ]; then
```

Shellcheck (0.11.0)

[warning] Unknown binary operator.

(SC2057)